lib/src/main/java/com/auth0/spring/security/api/JwtAccessDeniedHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.auth0.spring.security.api;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.web.access.AccessDeniedHandlerImpl;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Custom handler for access denied exceptions.
+ */
+class JwtAccessDeniedHandler extends AccessDeniedHandlerImpl {
+
+    @Override
+    public void handle(HttpServletRequest request, HttpServletResponse response, AccessDeniedException accessDeniedException) throws IOException, ServletException {
```

Comment:
`AccessDeniedHandler` specifies that `handle` must be public, which is why the class as package-private but the method as public